examples/reacher_v2/utils.py

```diff
@@ -7,11 +7,11 @@
 
 import numpy as np
 
-from algorithms.common.abstract.her import AbstractHER
-from algorithms.common.abstract.reward_fn import AbstractRewardFn
+from algorithms.common.abstract.her import HER as AbstractHER
```

Comment:
Why HER's class name is declared as AbstractHER, while RewardFn is not AbstractRewardFn?